instrumentation/sidekiq/lib/opentelemetry/instrumentation/sidekiq/middlewares/client/tracer_middleware.rb

```diff
@@ -18,6 +18,8 @@ def call(_worker_class, job, _queue, _redis_pool)
               attributes = {
                 SemanticConventions::Trace::MESSAGING_SYSTEM => 'sidekiq',
                 'messaging.sidekiq.job_class' => job['wrapped']&.to_s || job['class'],
+                'messaging.sidekiq.retry' => job['retry'].to_s,
```

Comment:
Is this a Numeric value? Would this be more interesting to keep as an `Integer`?

Reply:
Yes, it is. I agree with you. I will update it.